src/main/java/at/hannibal2/skyhanni/features/misc/halloweenlobbywaypoints/BasketWaypoints.kt

```diff
@@ -0,0 +1,61 @@
+package at.hannibal2.skyhanni.features.misc.halloweenlobbywaypoints
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.test.GriffinUtils.drawWaypointFilled
+import at.hannibal2.skyhanni.utils.LorenzColor
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.RenderUtils.drawDynamicText
+import net.minecraft.client.Minecraft
+import net.minecraftforge.client.event.RenderWorldLastEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class BasketWaypoints {
+    private val config get() = SkyHanniMod.feature.misc.halloweenBasket
+    private var waypoint: LorenzVec? = null
+    private var waypointName: String? = null
+
+    @SubscribeEvent
+    fun onRenderWorld(event: RenderWorldLastEvent) {
+        if (LorenzUtils.inSkyBlock || !LorenzUtils.onHypixel) return
```

Comment:
How about checking the sidebar the same way we do for SkyBlock, just for the lobbies? This logic feels like a slippery slope. Without the checks below, this call would be valid everywhere on Hypixel except SkyBlock.

Reply:
For the sake of consistency, I thought it would be cool to have a method like `LorenzUtils.inLobby()`. Since this is the first lobby feature, this would help write more lobby only features in the future.

Reply:
something something "i walk so others can run" 😅

Reply:
ok u have fun with that, the helper method i wrote is specifically for halloween also the most consistent general purpose solution might be locraw

Reply:
> also the most consistent general purpose solution might be locraw
Why request additional data when Hypixel already provides us with the sidebar data :(

Reply:
> > also the most consistent general purpose solution might be locraw
> 
> Why request additional data when Hypixel already provides us with the sidebar data :(

do i need to remind you of the `skiblock` incident